src/components/Screen/index.js

```diff
@@ -17,12 +17,12 @@ const Split = ({
 
     <div className='flex-ns'>
 
-      <main className={`mr4-ns ${aside ? 'w-70-ns' : 'w-100-ns'}`}>
+      <main className={`${aside ? 'w-70-ns mr4-ns' : 'w-100-ns'}`}>
```

Comment:
This fixes the extra margin on desktop that shouldn't be there for full screen main instances